mkdocs_static_i18n/plugin.py

```diff
@@ -118,9 +112,7 @@ def _get_non_translated_page(self, page, page_lang, config):
             )
             i18n_page.url = page.url.replace(page.name, i18n_page.name) or "."
         else:
-            i18n_page.dest_path = page.dest_path.replace(str(Path(f"{page.name}/")), "")
-            i18n_page.abs_dest_path = page.abs_dest_path.replace(str(Path(f"{page.name}/")), "")
-            i18n_page.url = page.url.replace(str(Path(f"{page.name}/")), "") or "."
+            i18n_page = self._fix_dest_paths(page)
 
         return i18n_page
```

Comment:
```suggestion
        return self._fix_dest_paths(i18n_page)
```

I guess it's better and safer to fix the object on return please so we're not stuck in a if / else context.